packages/mui-base/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -246,53 +240,49 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
         const percent = valueToPercent(value, min, max);
         const style = axisProps[axis].offset(percent);
 
-        const ValueLabelComponent = valueLabelDisplay === 'off' ? Forward : ValueLabel;
-
         return (
-          <React.Fragment key={index}>
-            <ValueLabelComponent
-              {...(!isHostComponent(ValueLabelComponent) && {
-                valueLabelFormat,
-                valueLabelDisplay,
-                value:
-                  typeof valueLabelFormat === 'function'
-                    ? valueLabelFormat(scale(value), index)
-                    : valueLabelFormat,
-                index,
-                open: open === index || active === index || valueLabelDisplay === 'on',
-                disabled,
-              })}
-              {...valueLabelProps}
-              className={clsx(classes.valueLabel, valueLabelProps.className)}
-            >
-              <Thumb
-                data-index={index}
-                data-focusvisible={focusedThumbIndex === index}
-                {...thumbProps}
-                className={clsx(classes.thumb, thumbProps.className, {
-                  [classes.active]: active === index,
-                  [classes.focusVisible]: focusedThumbIndex === index,
+          <Thumb
+            key={index}
+            data-index={index}
+            data-focusvisible={focusedThumbIndex === index}
+            {...thumbProps}
+            className={clsx(classes.thumb, thumbProps.className, {
+              [classes.active]: active === index,
+              [classes.focusVisible]: focusedThumbIndex === index,
+            })}
+            style={{
+              ...style,
+              pointerEvents: disableSwap && active !== index ? 'none' : undefined,
+              ...thumbProps.style,
+            }}
+          >
+            <Input
+              data-index={index}
+              aria-label={getAriaLabel ? getAriaLabel(index) : ariaLabel}
+              aria-valuenow={scale(value)}
+              aria-labelledby={ariaLabelledby}
+              aria-valuetext={
+                getAriaValueText ? getAriaValueText(scale(value), index) : ariaValuetext
+              }
+              value={values[index]}
+              {...inputProps}
+            />
+            {ValueLabel ? (
+              <ValueLabel
+                {...(!isHostComponent(ValueLabel) && {
+                  valueLabelFormat,
+                  index,
+                  disabled,
                 })}
-                style={{
-                  ...style,
-                  pointerEvents: disableSwap && active !== index ? 'none' : undefined,
-                  ...thumbProps.style,
-                }}
+                {...valueLabelProps}
+                className={valueLabelProps.className}
```

Comment:
```suggestion
                className={clsx(classes.valueLabel, valueLabelProps.className)}
```
Or if we no longer have `classes.valueLabel`, just drop the prop, it is already included in `valueLabelProps`.